pkg/test/application/echo/client/main.go

```diff
@@ -95,13 +95,11 @@ func main() {
 	opts, err := newBatchOptions()
 	if err != nil {
 		log.Fatalf("Failed to setup client: %v", err)
-		os.Exit(1)
```

Comment:
Does fatal exit these days? I remember the argument was not to exit.

Reply:
No, it doesn't exit. GoLand claimed it did and I assumed it was right. I reverted these changes.